index.js

```diff
@@ -193,7 +195,8 @@ const createFocusTrap = function (elements, userOptions) {
         //  web components if the `tabbableOptions.getShadowRoot` option was used for
         //  the trap, enabling shadow DOM support in tabbable (`Node.contains()` doesn't
         //  look inside web components even if open)
-        tabbableNodes.find((node) => node === element)
+        tabbableNodes.find((node) => node === element) ||
+        composedPath?.includes(container)
```

Comment:
Should the `includes` test be first since it may be faster than the `find`?

Reply:
Yes, actually. In our very initial approach, this order would've been better, but thanks to your `composedPath` idea, I think you're right. The composed path test should be second, and then `find()` be third.

Reply:
done!